src/repositories/seekers.js

```diff
@@ -32,24 +26,13 @@ const findByUuid = async (where = {}) => {
   return await baseRepo.first("seekers", where);
```

Comment:
I think we should keep the check for deleted users by default. Incase the developer forget to pass it in the where clause?

Reply:
This deletedAt: null check is already in `baseRepo.first()` method by default @ShubhamJoshi-1994